common/BUILD.gn

```diff
@@ -43,7 +43,18 @@ source_set("channel_info") {
   }
 }
 
+source_set("common_utils") {
+  sources = [
+    "brave_common_utils.cc",
+    "brave_common_utils.h",
+  ]
+
+  deps = [ "//base" ]
+}
+
 group("common") {
+  deps = [ ":common_utils" ]
```

Comment:
In chromium, module that shared from browser and renderer both is added to `common` directory.
Instead, I think adding util files under `brave/browser/ui` is more appropriate ad that api is only used in ui module.

Reply:
Okay.

Just curious, where would you suggest placing this if we want to use it in the `component` directory?

Reply:
Well.. if it's shared with component layer also, I think we can have new folder under `brave/components/`.

Reply:
Okay, Thanks!